packages/build_packages.sh

```diff
@@ -8,12 +8,13 @@ set -ex
 # Change to packages directory.
 cd "$(dirname "$0")"
 
-# ensure ordered by dependency dag
+# Ensure ordered by dependency dag
 while read -r package_dir; do
-    if [ -z "$package_dir" ]; then
-        # Skip empty lines
-        continue
-    fi
+    # Ignore empty lines
+    [[ -z "$package_dir" ]] && continue
```

Comment:
Can we not using unnecessary bashisms (i.e. `[[`) and keep the longer but more readable if/then/fi ?

Reply:
@nsoranzo do you mind keeping `[[` for the sake of its convenient `=~` operator? I've combined it with if/then/fi for readability. Or would you prefer to drop `[[` because it is bash-specific?

Reply:
I'd just like to keep the bashisms at the minimum, so it's fine to use `[[ ... ~= ]]`, but maybe restore the single `[` (and quoting) for `[ -z "$package_dir" ]`, if that makes sense?